src/parsing/parser.cc

```diff
@@ -504,31 +535,32 @@ void MaybeResetCharacterStream(ParseInfo* info, FunctionLiteral* literal) {
   info->ResetCharacterStream();
 }
 
-void MaybeProcessSourceRanges(ParseInfo* parse_info, Expression* root,
-                              uintptr_t stack_limit_) {
-  if (root != nullptr && parse_info->source_range_map() != nullptr) {
-    SourceRangeAstVisitor visitor(stack_limit_, root,
-                                  parse_info->source_range_map());
-    visitor.Run();
-  }
-}
+// void MaybeProcessSourceRanges(ParseInfo* parse_info, Expression* root,
```

Comment:
compiler seems to think this is unused. any idea why?

Reply:
Maybe it can't guarantee that it's used now that it doesn't know what type stack_limit_ is? Why did you start accessing stack_limit_ off of impl() since it's declared in ParserBase?